README.md

```diff
@@ -0,0 +1,281 @@
+
+
+
+Now that our Function App exists, we can create an actual function. 
+
+![createNewFunction](images/createNewFunction.png)
+
+Your functions page should look something like this, but empty. To create a function, click the **+Add** button and select HTTP trigger. Set your authorization level as function- this means that the caller needs to provide a key.
+
+![newHTTPFunction](images/newHTTPFunction.png)
+
+
+
+Wait for the function to deploy, then click into the **Code + Test** tab and go to your index.js file.
+
+Replace the code with this: 
+
+```js
+var multipart = require("parse-multipart");
+var request = require('request-promise');
+var util = require('util');
+ 
+module.exports = async function (context, req) {
+    context.log('JavaScript HTTP trigger function processed a request.'); 
+    // encode body to base64 string
+    var bodyBuffer = Buffer.from(req.body);
+    
+    var boundary = multipart.getBoundary(req.headers['content-type']);
+    // parse the body with parse-multipart
+    var parts = multipart.Parse(bodyBuffer, boundary);
+    
+  	//calls a function that calls the Face API
+    var result = await analyzeImage(parts[0].data);
+
+  	//stick the data returned into our http response!
+    context.res = {
+        body: {
+            result
+        }
+    };
+    console.log(result); 
+    context.done();  
+};
+ 
+async function analyzeImage(byteArray){
+    
+    const subscriptionKey = 'YOUR KEY';
+    const uriBase = 'YOUR ENDPOINT' + '/face/v1.0/detect';
+
+  //Face API params
+    const params = {
+        'returnFaceId': 'true',
+        'returnFaceAttributes': 'emotion'
+    };
+
+    const options = {
+        uri: uriBase,
+        qs: params,
+        body: byteArray,
+        headers: {
+            'Content-Type': "application/octet-stream",
+            "Ocp-Apim-Subscription-Key": subscriptionKey
+        }
+    }
+    let jsonResponse;
+    
+    await request.post(options, (error, response, body) => {
+        if (error){
+            console.log('Error: ' + error);
+            return;
+        }
+
+        jsonResponse = JSON.parse(body);   
+    });
+    return jsonResponse;
+    
+}
+```
+
+This code is triggered by an HTTP request and parses HTML multipart data(our image!) with the help of the extremely helpful [parse-multipart](https://www.npmjs.com/package/parse-multipart) library. We're then calling the Face API in the function **analyzeImage** using the subscription key and endpoint from earlier, passing in our parsed image. This uses the [request-promise](https://www.npmjs.com/package/request-promise) library, which is basically request but with Promise support, which we require in our async **analyzeImage** function. Using `JSON.parse()`, we return the body in a JSON format. 
+
+
```

Comment:
Can you add an example of a pic you ran here and talk about what you got back? was it accurate? describe the results you got and takl about if it was something you expected.